kur/supplier/speechrec.py

```diff
@@ -516,11 +550,13 @@ def __init__(self, url=None, path=None, checksum=None, unpack=None,
 		self.downselect(samples)
 
 		logger.debug('Creating sources.')
+		data_cpus = multiprocessing.cpu_count() - 1 if not data_cpus else data_cpus
```

Comment:
If the machine is a single-core machine, then then will cause data_cpus to be set to zero. Shouldn't it be something like:

    data_cpus = max(1, multiprocessing.cpu_count() - 1 if not data_cpus else data_cpus)